README.md

```diff
@@ -48,19 +48,17 @@ If modifying existing spells which haven't been migrated to DuneSQL yet, it is r
 
 Steps to migrate:
 - Find the spell SQL file to translate in the repo
-  - rename existing spark SQL file to append `_legacy.sql` at the end and keep the syntax as-is within the spell
-  - modify the `alias` property in the config block to leverage the new alias dbt macro, which includes extended parameters passed in
-    - from `alias = 'blocks',` to --> `alias = alias('blocks', legacy_model=True),`
-- Update all the references downstream of modified spell(s) to reference new legacy spells in step 1
-  - for example, any instance of `{{ ref( ) }}` downstream of modified spell(s)
-  - to find downstream spells, the following can be run: `dbt ls --resource-type model --output name --select <insert spell name>+`
-  - **future note:** when downstream spells are also migrated, we will be able to revert back to reference DuneSQL versioned spells. We will be working in a upstream --> downstream lineage path to full migration.
-- Duplicate the `_legacy.sql` file in your local repo & then remove the `_legacy` suffix
+  - **note:** you can safely ignore the _legacy file, as they are intended to maintain the spark syntax (unless you need to alter logic in spark spell for bug fixes)
+    - previously, it was requested for users to generate the _legacy files in the migration process, but the Dune team has gone ahead and universally built the legacy files for simplicity in the process of migration
```

Comment:
So I do not need to generate `*_legacy.sql` files in future migration PRs?

Reply:
this is correct, they should already be there.